infra/fendermint/scripts/genesis.toml

```diff
@@ -31,6 +31,7 @@ extend = "fendermint-tool"
 env = { "ENTRY" = "fendermint", "CMD" = """genesis --genesis-file /data/genesis.json ipc gateway --subnet-id /r0 \
     --bottom-up-check-period 10 \
     --msg-fee 10 \
+    --min-collateral 1000000000000000000 \
```

Comment:
It's strange that you need this because the [smoke tests](https://github.com/consensus-shipyard/ipc-monorepo/blob/main/fendermint/testing/smoke-test/scripts/init.sh#L57-L64) don't use it and this [option doesn't exist any more](https://github.com/consensus-shipyard/ipc-monorepo/blob/main/fendermint/app/options/src/genesis.rs#L128-L147). It looks like it was removed in https://github.com/consensus-shipyard/ipc-monorepo/commit/b569f87b189aa0f16dbbe6aac7268a75a6ffcc6b#diff-7db55b0afa66369cda521e35945962d2798ead266a495a3e25235f89387f75e9

Reply:
I see - i think that when `cargo make --makefile ./infra/fendermint/Makefile.toml testnode` is run, it pulls a docker image from the old fendermint repo, not the mono repo. I am still investigating to understand and have opened a new issue, since this does not seem to be the correct bugfix:

https://github.com/consensus-shipyard/ipc/issues/546